utils/ClosedCaption/DFXPParser.as

```diff
@@ -298,12 +364,14 @@ package
 					
 		private function debugLog(msg:String):void
 		{
-
+			ExternalInterface.call("console.log"," "+msg);

```

Comment:
@angieisrani  This is called only in the error scenarios to log the error and will be logged irrespective of whether it is debug or release build.